lib/map.go

```diff
@@ -12,3 +14,36 @@ type MapDataUpload struct {
 	IsForSale       []bool   `json:"IsForSale"`
 	BuyPrice        []int    `json:"BuyPrice"`
 }
+
+func (m *MapDataUpload) StringArrays() [][]string {
+	result := [][]string{}
+	for i := range m.BuildingType {
+		result = append(result, []string{
+			fmt.Sprintf("%d", m.ZoneID),
+			fmt.Sprintf("%d", m.BuildingType[i]),
+			fmt.Sprintf("%d", m.AvailableFood[i]),
+			fmt.Sprintf("%d", m.Reward[i]),
+			fmt.Sprintf("%d", m.AvailableSilver[i]),
+			m.Owners[i],
+			fmt.Sprintf("%t", m.Buildable[i]),
+			fmt.Sprintf("%t", m.IsForSale[i]),
+			fmt.Sprintf("%d", m.BuyPrice[i]),
+		})
+	}
+
+	return result
+}
+
+func MapDataHeaders() []string {
```

Comment:
We need to be more consistent in naming this function.

Can we just use the `json` tag from the struct instead of rewriting the same thing?

Reply:
The first, yeah, will rename the others.

The second not sure how.